browser/src/Services/Language/LanguageClientStatusBar.tsx

```diff
@@ -9,7 +9,10 @@ import * as React from "react"
 
 import * as Oni from "oni-api"
 
+import styled from "styled-components"
+
 import { Icon } from "./../../UI/Icon"
+import { configuration } from "./../Configuration"
```

Comment:
Using the global configuration object is actually what is causing the test failure. We pass a `MockConfiguration` to the LanguageManager, but that doesn't get passed to the status bar object here. Instead of using the global configuration object, we should pass in a `Configuration` to the constructor here. That will help insulate this class from global state, and get the test to pass 😄